### ISSUE TITLE
menu-button doesn't compile with Typescript

### ISSUE THREAD

**Issue by brookback** [reach/reach-ui#405]
## 🐛 Bug report

1. `npm install --save @reach/menu-button`.
2. Write TSX code that imports from `@reach/menu-button`.

### Current Behavior

It doesn't compile with Typescript:

```bash
node_modules/@reach/menu-button/index.d.ts:11:33 - error TS2307: Cannot find module 'styled-components'.

11 import { StyledComponent } from "styled-components";
                                   ~~~~~~~~~~~~~~~~~~~

[15:05:32] Found 1 error. Watching for file changes.
```

### Expected behavior

It compiles with Typescript.

### Reproducible example

https://codesandbox.io/s/hardcore-kapitsa-v1c58

App doesn't load.

### Suggested solution(s)

This commit seems problematic: https://github.com/reach/reach-ui/commit/18281e8be79e2cf2399408b835e5f0879f657592. It adds an import for `styled-components` inside of `packages/menu-button/index.d.ts`, even though:

- It doesn't seem to be used.
- It doesn't exist in `menu-button` package's `package.json`.

### Additional context

-

### Your environment

| Software                 | Name/Version(s) |
| ------------------------ | --------------- |
| **menu-button** |0.6.3
| React                    |16.8.6
| Browser                  |-
| Assistive tech           |Typescript 3.7.2
| Node                     |8.12.0
| npm/yarn                 |6.4.1
| Operating System         |MacOS

**Comment by hipstersmoothie** [comment]
I am seeing this too. I agree that `styled-components` is probably not needed in the .d.ts file since it doesn't seem to be used

**Comment by chancestrickland** [comment]
Oof, my bad on this one. I'll get this patched ASAP!

**Comment by chancestrickland** [comment]
Fixed in 0.6.4

**Comment by brookback** [comment]
Thanks!
